analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -444,6 +444,8 @@ private static void VerifyCode<TRunner>(string code, ProjectType projectType, Im
 
         private class TestSERunnerCS : CS.SymbolicExecutionRunner
         {
+            public TestSERunnerCS() : base(AnalyzerConfiguration.AlwaysEnabled) { }
```

Comment:
Some tests above depend on S2259 and will give us some trouble later, until the rule will be able to recognize the simple cases.